packages/core/data-transfer/lib/providers/local-file-source-provider/index.ts

```diff
@@ -49,6 +49,8 @@ class LocalFileSourceProvider implements ISourceProvider {
 
   options: ILocalFileSourceProviderOptions;
 
+  #metadata?: IMetadata;
```

Comment:
This is my main question. Should we store it here, or just read it twice? IMO we're fine to store it, since this is the one file that is slurped in instead of streamed.

Reply:
Make sense to have this here, lgtm